.github/workflows/post-release-changes.yml

```diff
@@ -46,7 +92,7 @@ jobs:
           title: Post release changes
           branch: releas-branch
           delete-branch: true
-          base: ${{ github.event.inputs.BASE }}
+          base: ${{ inputs.BASE }}
```

Comment:
base branch should be from inputs na.

Reply:
babe branch taking from user input only and that is present in release-script caller workflow